x-pack/platform/plugins/shared/lens/public/datasources/form_based/datapanel.scss

```diff
@@ -2,6 +2,14 @@
   width: 100%;
   height: 100%;
   padding: $euiSize $euiSize 0;
+  .unifiedFieldListItemButton.kbnFieldButton {
+    background: none;
+    box-shadow: none;
+    margin-bottom: calc($euiSizeXS / 2);
+  }
+  .unifiedFieldListItemButton__dragging {
+    background: $euiColorEmptyShade;
```

Comment:
Should this color be changed to `$euiColorBackgroundBasePlain`?

Reply:
done in [49d8096](https://github.com/elastic/kibana/pull/204839/commits/49d8096ab969b7f4b1a834b4ebba60b373c80808)